flask_appbuilder/models/sqla/base.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.engine import Connection, Engine
+from sqlalchemy.sql.schema import Table
+
+
+class SQLA(SQLAlchemy):
+    """
+    This is a child class of flask_SQLAlchemy
+    It's purpose is to override the declarative base of the original
+    package. So that it is bound to F.A.B. Model class allowing the dev
+    to be in the same namespace of the security tables (and others)
+    and can use AuditMixin class alike.
+
+    Configure just like flask_sqlalchemy SQLAlchemy
+    """
+
+    def init_app(self, app: Flask) -> None:
+        session_options = app.config.get("SQLALCHEMY_SESSION_OPTIONS", {})
+        if session_options:
+            self.session = self._make_scoped_session(session_options)
```

Comment:
Hey @dpgaspar - we are testing the alpha version of FAB 5 for Airlow and we think this one gives us a hard time running our tests.

Basically in our tests we are creating a number of FabAppBuilders  (and we override some parts of it) - but we have an issue that when we access a session in pytest (and we run `init_app` multiple times in the same interpreter - in pytest - every time with a different App, the session that gets created here stays connected - we have > 90 connections to PostgresDB while running tests and we have very hard time on tracking where and how they should be closed. 

Does it ring a bell?